### ISSUE TITLE
Compilation fails against  http-client >= 0.5.0

### ISSUE THREAD

**Issue by danidiaz** [kallisti-dev/hs-webdriver#98]
webdriver has problems compiling against versions of [http-client](http://hackage.haskell.org/package/http-client) >= 0.5.0. It gives the following error:

```
src\Test\WebDriver\Internal.hs:16:82: error:
    Module
    `Network.HTTP.Client'
    does not export
    `HttpException(ResponseTimeout)'
```

It seems the latest versions of http-client made changes to the HttpException datatype.

**Comment by snoyberg** [comment]
This is now affecting Stackage as well.

**Comment by kallisti-dev** [comment]
Should be fixed by #96. I need to fix problems with travis config before I can verify.

**Comment by kallisti-dev** [comment]
Travis builds have been fixed as you can see [here](https://travis-ci.org/kallisti-dev/hs-webdriver/builds/164044707). I'll close this issue and ship this fix with the next release.
